vendor/github.com/docker/docker/api/types/stats.go

```diff
@@ -120,7 +120,7 @@ type NetworkStats struct {
 	RxBytes uint64 `json:"rx_bytes"`
 	// Packets received. Windows and Linux.
 	RxPackets uint64 `json:"rx_packets"`
-	// Received errors. Not used on Windows. Note that we dont `omitempty` this
+	// Received errors. Not used on Windows. Note that we don't `omitempty` this
```

Comment:
This file is vendored, so you shouldn't change it here.

Looks like it was recently fixed upstream, so we should pick up this fix when we update the dependency: https://github.com/moby/moby/blob/master/api/types/stats.go